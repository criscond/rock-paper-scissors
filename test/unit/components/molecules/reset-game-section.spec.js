import resetGameSection from '../../../../src/js/components/molecules/reset-game-section';
describe('Reset Game Section', () => {

	it('should create a section', () => {
		expect(resetGameSection()).toMatchSnapshot();
	});

});
