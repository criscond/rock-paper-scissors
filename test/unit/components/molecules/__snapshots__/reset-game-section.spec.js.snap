// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset Game Section should create a section 1`] = `
<section
  class="column"
>
  <button
    id="resetGame"
    type="button"
  />
</section>
`;
